legion/engine/llri-dx/detail/command_group.cpp

```diff
@@ -67,17 +67,17 @@ namespace LLRI_NAMESPACE
     
     result CommandGroup::impl_allocate(const command_list_alloc_desc& desc, uint8_t count, std::vector<CommandList*>* cmdLists)
```

Comment:
in this case i'd say count should've actually also been a size_t

Reply:
there's no practical reason to have one over the other but I indicate a maximum of 256 commandlists per commandgroup which I think is reasonable

Reply:
well technically it causes an unnecessary cast on line 78. but oh well, tbh idrc XD